### PR TITLE
Pagetree adaptions after pair review

### PR DESCRIPTION
**Pull request by FinalAngel**

### CODE REVIEW

File: cms/static/cms/js/modules/cms.pagetree.js [hunk 1]
```diff
@@ -277,6 +278,8 @@ var CMS = window.CMS || {};
             // set event for cut and paste
             this.ui.container.on(this.click, '.js-cms-tree-item-cut, .js-cms-tree-item-copy', function (e) {
                 e.preventDefault();
+                // we need to cache the node and type so `_toggleHelpers`
+                // wil trigger the correct behaviour
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.pagetree.js]: `wil` typo

File: cms/static/cms/js/modules/cms.pagetree.js [hunk 2]
```diff
@@ -388,12 +392,15 @@ var CMS = window.CMS || {};
...
+                // we only need to move the node when not using drag & drop
+                // for example when we use copy & paste
+                // jstrees drag & drop will move the node for us
                 if (that.cache) {
                     that.ui.tree.jstree('move_node',
                         that.ui.tree.find('li[data-id="' + obj.id + '"]'),
                         that.ui.tree.find('li[data-id="' + obj.target + '"]'));
                 }
-                that.cache = undefined;
+                that.cache = null;
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.pagetree.js]: not much of a difference but maybe it makes sense to move this statement into an if

Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.pagetree.js]: why into a statement? I basically need to reset it afterwards

File: cms/static/cms/js/modules/cms.pagetree.js [hunk 3]
```diff
@@ -480,7 +485,8 @@ var CMS = window.CMS || {};
             var parent = this.ui.tree.jstree('get_parent', element);
             var nextDom = this.ui.tree.jstree('get_next_dom', element, true);
             var prevDom = this.ui.tree.jstree('get_prev_dom', element, true);
-            var parentDom = this.ui.tree.jstree('get_node', parent);
+            // this refers to the parent jstree node which is the tree itself
+            var parent = this.ui.tree.jstree('get_node', parent);
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.pagetree.js]: you already have `parent` variable on line 485

File: cms/static/cms/js/modules/cms.pagetree.js (hunk 2 above)
Comment (vxsx) [inline cms/static/cms/js/modules/cms.pagetree.js]: only if it was something else before, no?

**Comment by vxsx** [comment]
:+1:

**Pull request merged**